packages/bpk-component-spinner/src/BpkLargeSpinner.js

```diff
@@ -26,14 +28,18 @@ import SPINNER_TYPES from './spinnerTypes';
 
 const getClassName = cssModules(STYLES);
 
-const BpkLargeSpinner = props => {
+type Props = {
+  type: string,
```

Comment:
Same comment as earlier about using `$Keys<typeof SPINNER_TYPES>`.

Reply:
Did this suggestion and seems to work fine